### ISSUE TITLE
Feedback: "Navigate out of the start of a radio group in reading mode" (Radio Group Example Using aria-activedescendant, Test 17)

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#922]
## Description of Behavior

JAWS does not announce the boundary of the radio group when tabbing backward out of the group to the previous link.

I thought we discussed this test with Glen and had agreed that we would either remove the test or make it optional. My opinion is that it should be removed, i.e., it would not be desirable behavior to announce exit from a radio group when tabbing.

## Test Setup

- Test File: [test-17-navigate-out-of-the-start-of-a-radio-group-reading-jaws.collected.html](https://aria-at.netlify.app/tests/radiogroup-aria-activedescendant/test-17-navigate-out-of-the-start-of-a-radio-group-reading-jaws.collected.html)
- AT: JAWS (version undefined)
- Browser: Chrome (version undefined)

**Comment by mcking65** [comment]
@jscholes 

I gave this more thought after our discussion yesterday. I think there is a simple solution: Remove the commands Tab and Shift+Tab from this test. That is all we have to do.

While it is possible to navigate out of a radio group with Tab or Shift+Tab, there are no specific behaviors relevant to radio group that need to be tested. This is generally true when navigating away from any element with Tab. The screen reader behaviors that need to be tested are related to the element that receives focus, not the element that lost focus.

Thus, we need only remove those commands from this test and from test 20 (issue #923), and we do not need to add any new tests to the plan.

What do you think?

**Comment by jscholes** [comment]
@mcking65 I think this requires further discussion.  All of our test plans include evaluation of screen reader behaviour while moving focus in reading mode, and I don't feel comfortable making a single test, within a single test plan, an outlier in that regard.

In my experience, screen reader behaviour can be different, and/or problematic, when focus is moved in reading mode versus interaction mode.  For example, only partial reading of the target element, and the wrong focus position entirely, are two issues I've encountered.

It could be said that, by only testing a single element type as the focus target (a link), we aren't fully exercising the possibilities in this area regardless, and that would be a factual statement.  But it would also apply to other areas of ARIA-AT tests, where we test what we are feasibly able, given the need for human test runs.

If we want to move away from testing screen reader behaviour once focus is moved away from the specific type of element(s) under test, that should be an explicit decision with Community Group input, as it would significantly change our approach to testing.  I don't believe it's something we should do for one single test to solve issues of logistics within the ARIA-AT app.

**Comment by mcking65** [comment]
@jscholes 

I am not advocating removing the test. We would still test navigating out of the group with all commands where there is some screen reader behavior relevant to the radio group.

I am suggesting that when tabbing away from an element, there is no screen reader behavior related to the element that is losing focus that we need to test. So, for the navigate away tests, the Tab key commands should not be included. Obviously, commands like arrow and ctrl-opt-arrow should be tested.

**Comment by jscholes** [comment]
Closed via https://github.com/w3c/aria-at/pull/928.
